qcengine/programs/tests/test_entos.py

```diff
@@ -3,7 +3,7 @@
 import qcelemental as qcel
 import qcengine as qcng
 from qcelemental.testing import compare_recursive
-from qcengine.testing import qcengine_records, using_entos
+from qcengine.testing import qcengine_records, using
```

Comment:
can we standardize btwn `using` and `testing.using`? I slightly favor former.

after going through the PR, it seems former is indeed favored. any reason it can't be universal? there's only a couple files with latter.

Reply:
I wanted to open an issue on this, but overall I think we need to standardize on `using` so that this strategy works for fixtures. I can spin out an issue to standardize this.

Reply:
sounds good. i'll standardize in p4 where we're borrowing using_* from qcng. though now that I think about it, that reform may be in the ddd branch.